docker-compose.yml

```diff
@@ -112,7 +112,7 @@ services:
     build:
       dockerfile: packages/hoppscotch-backend/Dockerfile
       context: .
-      target: prod
+      target: dev
```

Comment:
Is this intentional ?

Reply:
Yup same reason, only for dev. Will revert it before merging.